### PR TITLE
Clarify metrics and logging as part of spec SIG

### PR DESCRIPTION
**Pull request by mtwo**
Clarify that metrics and logging are workgroups under the specification SIG

### CODE REVIEW

**Comment by mtwo** [comment]
@jmacd FYI

File: README.md [hunk 1]
```diff
@@ -122,6 +122,22 @@ Dial by your location
 Meeting ID: 672 939 6170
 Find your local number: https://zoom.us/u/abrmtXWOh8
 
+#### Logs Sub-SIG
```
Comment (yurishkuro) [inline README.md:125]: perhaps "working group" instead of Sub-SIG

Comment (mtwo) [reply README.md:125]:
```suggestion
#### Logs Working Group
```

File: README.md [hunk 2]
```diff
@@ -122,6 +122,22 @@ Dial by your location
...
+
+The Logs sub-SIG of the specifiction SIG meets every other week on Wednesdays at alternating times via [Zoom](https://zoom.us/j/8203130519). Meeting notes are held in this [doc](https://docs.google.com/document/d/1cX5fWXyWqVVzYHSFUymYUfWxUK5hT97gc23w595LmdM/edit#).
+
+You can also join us on [gitter](https://gitter.im/open-telemetry/logs).
+
+#### Metrics Sub-SIG
```
Comment (mtwo) [inline README.md:131]:
```suggestion
#### Metrics Working Group
```

**Pull request merged**